cmd/prysmctl/testnet/generate_genesis.go

```diff
@@ -263,12 +263,16 @@ func generateGenesis(ctx context.Context) (state.BeaconState, error) {
 		// set timestamps for genesis and shanghai fork
 		gen.Timestamp = f.GenesisTime
 		gen.Config.ShanghaiTime = interop.GethShanghaiTime(f.GenesisTime, params.BeaconConfig())
+		//gen.Config.CancunTime = interop.GethCancunTime(f.GenesisTime, params.BeaconConfig())
+		gen.Config.CancunTime = interop.GethCancunTime(f.GenesisTime, params.BeaconConfig())
 		log.
 			WithField("shanghai", gen.Config.ShanghaiTime).
+			WithField("cancun", gen.Config.CancunTime).
 			Info("setting fork geth times")
 		if v > version.Altair {
 			// set ttd to zero so EL goes post-merge immediately
 			gen.Config.TerminalTotalDifficulty = big.NewInt(0)
+			gen.Config.TerminalTotalDifficultyPassed = true
```

Comment:
Once we have updated our e2e geth dependency to the v1.12 release, this should be moved into `runtime/interop/genesis.go`. This is because geth 1.12 won't run in miner mode and this flag needs to be specified for it to startup.